pollination/ladybug_comfort/map.py

```diff
@@ -291,14 +291,21 @@ class UtciMap(Function):
         'will be for the entire result_sql run period.', default=''
     )
 
+    plain_text = Inputs.str(
+        description='Flag to note whether the output should be formatted as a '
+        'plain text CSV or whether it should be formatted as a binary numpy '
+        'array.', default='plain-text',
```

Comment:
I think we should add a sentence to indicate why the user should pick one over the other. Not everyone would know the difference between the two, and the right choice for their case.

Reply:
@mostaphaRoudsari, I expanded the description of the input.